### PR TITLE
Template css corrected and ready for merging with development branch

### PR DESCRIPTION
**Pull request by ethanedwards115**

### CODE REVIEW

**Review by finlaysaunders — APPROVED** [review]
:)

File: pages/template.html [hunk 1]
```diff
@@ -3,16 +3,16 @@
 
 <head>
   <meta charset="utf-8">
-  <link rel="stylesheet" href="../css/template.css">
-
   <link rel="stylesheet" href="../css/bootstrap.css">
+  <link rel="stylesheet" href="../css/master.css">
+  <link rel="stylesheet" href="../css/template.css">
```
Comment (Arcfej) [inline pages/template.html:8]:
```suggestion
```

**Review by Arcfej — CHANGES_REQUESTED** [review]
line needs to removed than ok

File: pages/template.html (hunk 1 above)
Comment (ethanedwards115) [reply pages/template.html:8]: Okay, will be removed now

**Pull request merged**